pkg/async-gateway/exit/exit.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2021 Cortex Labs, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package exit
```

Comment:
isn't there an exit package already?

Reply:
Yes there is. We have a ticket to address this https://app.zenhub.com/workspaces/cortex-5fbdec008baa5b0010888837/issues/cortexlabs/internal/194.

Reply:
We are removing the exit package for the gateway. The only function will move to main. We also need to update the internal ticket.